app/editor/src/components/breadcrumb/Breadcrumb.tsx

```diff
@@ -9,26 +8,20 @@ export interface IBreadcrumbProps {
   /** the current path being viewed in the clip explorer */
   path: string;
   /** set path of the parent component */
-  setPath: (path: string) => void;
+  onNavigate: (path: string) => void;
 }
 
 /** component used to navigate state controlled path directory, also updates the
  *  search params to the current directory
  */
-export const Breadcrumb: React.FC<IBreadcrumbProps> = ({ label, path, setPath }) => {
-  const [, setSearchParams] = useSearchParams();
-
+export const Breadcrumb: React.FC<IBreadcrumbProps> = ({ label, path, onNavigate }) => {
   if (path === '') path = '/';
 
-  React.useEffect(() => {
```

Comment:
This was causing a lot of grief and a bad user experience.  Now the breadcrumb is not coupled with the URL.